### PR TITLE
feat: granular thorchain lending flags

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box - note, savers and finally swapper will follow this in the stack.

The reason for this order is swaps and LP are the main primitives of THORfi. The naive implementation is to simply be granular per feature, but more research will need to be made on how to simplify things, which will be done in the final swapper PR i.e things could most likely simplified as enabling thorchain interactions (`MsgDeposit` and the like) and pools interactions (all swaps, LP, savers).

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

- contributes to https://github.com/shapeshift/web/issues/6520

> What protocols, transaction types or contract interactions might be affected by this PR?

Low - this should come as no change for THORCHain Lending

### Engineering

- THORChain Lending pools are shown as halted with the flags off

### Operations

- THORChain Lending pools show no regressions against develop

## Screenshots (if applicable)

- Lending borrows and repayments disabled

image
image

- Lending borrows and repayments enabled

image
image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/shapeshift/web/6532?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#6532** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6532?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#6527** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6527?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/pages/Lending/Pool/components/Repay/RepayInput.tsx [hunk 1]
```diff
@@ -338,6 +341,7 @@ export const RepayInput = ({
   }, [_isSmartContractAddress])
 
   const quoteErrorTranslation = useMemo(() => {
+    if (!isThorchainLendingRepayEnabled) return 'trade.errors.tradingNotActiveNoAssetSymbol'
```
Comment (kaladinlight) [inline src/pages/Lending/Pool/components/Repay/RepayInput.tsx:344]: Might as well make a new translation for this one. Feels weird showing trades temp halted on repayment.

suggestion: `Repayments are temporarily halted.`

Comment (gomesalexandre) [reply src/pages/Lending/Pool/components/Repay/RepayInput.tsx]: b8c43b0ad4 - added some lending-specific copies for borrowing too while at it 

image
image

File: src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx [hunk 2]
```diff
@@ -427,7 +430,8 @@ export const BorrowInput = ({
   ])
 
   const quoteErrorTranslation = useMemo(() => {
-    if (isHardCapReached) return 'defi.modals.saversVaults.haltedTitle'
+    if (isHardCapReached) return 'lending.errors.borrowingHalted'
+    if (!isThorchainLendingBorrowEnabled) return 'lending.errors.borrowingDisabled'
```
Comment (kaladinlight) [inline src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx:434]: i would still have disabled as highest priority imo

**Review by kaladinlight — APPROVED** [review]

File: src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx (hunk 2 above)
Comment (gomesalexandre) [reply src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx:433]: Absolutely! 10991bd1e54bee512adf09a080983761b481539b

**Pull request merged**